src/_utils/branding.ts

```diff
@@ -2,7 +2,7 @@ interface Color {
   [propName: string]: string
 }
 
-const palette: Color = {
+export const palette: Color = {
```

Comment:
We already export the color const to use palette.

Reply:
Yeah. I'll use it for consistency's sake, but having `color.primaryText` isn't really explicit.